README.md

```diff
@@ -1,4 +1,4 @@
-# Async.js
+![Async Logo](https://raw.githubusercontent.com/caolan/async/master/logo/async-logo_readme.jpg)
```

Comment:
Pros: We could place easter eggs during christmas, and it would work everywere... ( \o/ )